packages/reakit-theme-default/package.json

```diff
@@ -48,7 +48,7 @@
     "eslint": "^5.9.0",
     "jest-cli": "^23.6.0",
     "opn-cli": "^4.0.0",
-    "reakit": "^0.15.11",
+    "reakit": "^0.15.12",
```

Comment:
This diff is really weird. Why is this showing changes that are already on master?

Reply:
The same happens to the other PRs